tests/resolvers/Query/actionItemsByOrganization.spec.ts

```diff
@@ -7,7 +7,7 @@ import { beforeAll, afterAll, describe, it, expect } from "vitest";
 import type mongoose from "mongoose";
 import { createTestActionItems } from "../../helpers/actionItem";
 import type { TestEventType } from "../../helpers/events";
-import { TestOrganizationType } from "../../helpers/userAndOrg";
+import type { TestOrganizationType } from "../../helpers/userAndOrg";
```

Comment:
Why is this change needed?

Reply:
These changes appeared on linting. Thought them as neccessary. Are they neccessary or shall I remove them?